backend/app/models.py

```diff
@@ -3,30 +3,37 @@
 from werkzeug.security import generate_password_hash, check_password_hash
 
 # create model classes here
-
-
-class Mentor:
-
-    # constructor
+class User:
+     # constructor
     def __init__(self, name, email):
         self.name = name
         self.email = email
 
     def serialize(self):
-        return {"name": self.name, "email": self.email}
+        return {
+            "name": self.name, 
+            "email": self.email
+            }
+
+class Mentor(User):
+    pass
 
 
-class Client:
+class Client(User):
 
     # constructor
-    def __init__(self, name, notes, attachments):
-        self.name = name
+    def __init__(self, name, email, notes, attachments):
+        super().__init__(name, email)
         self.notes = notes
         self.attachments = attachments
 
     def serialize(self):
         return {
             "name": self.name,
+            "email": self.email,
             "notes": self.notes,
             "attachments": self.attachments
         }
```

Comment:
What is the correct way to call serialize from User and add notes and attachments?

Reply:
If you have an instance of one of the subclasses of User (e.g. Client), then you can just do:
```
c = Client("name", "email", "notes", "attachments")
c.serialize()
```